lib/ava-files.js

```diff
@@ -262,13 +266,22 @@ class AvaFiles {
 			.filter(pattern => defaultIgnore.indexOf(pattern.split('/')[0]) >= 0)
 			.map(pattern => `!${pattern}`);
 
-		ignored = getDefaultIgnorePatterns().concat(ignored, overrideDefaultIgnorePatterns);
+		ignored = getDefaultIgnorePatterns().concat(
+			ignored,
+			overrideDefaultIgnorePatterns
+		);
 
 		if (paths.length === 0) {
-			paths = ['package.json', '**/*.js', '**/*.snap'];
+			paths.unshift('**/*.js');
+		}
+		if (paths.indexOf('package.json') === -1) {
+			paths.unshift('package.json');
+		}
+		if (paths.indexOf('**/*.snap') === -1) {
+			paths.unshift('**/*.snap');
 		}
 
-		paths = paths.concat(this.files);
+		paths.push(...this.files);
```

Comment:
We're targeting Node.js 4, which means spread isn't available 😞

Reply:
ok fixed